Dockerfile

```diff
@@ -18,18 +18,21 @@ ENV SHELL /bin/bash
 
 SHELL ["/bin/bash", "-c"]
 
-# Instal rust.
+# Install rust.
 RUN sudo apt-get -y install curl clang python3 python3-pip git
 RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
 ENV PATH="/home/erdos/.cargo/bin:${PATH}"
-RUN rustup default nightly
 
 # Get the erdos directory.
-RUN pip3 install setuptools setuptools-rust
 RUN mkdir -p /home/erdos/workspace
-RUN cd /home/erdos/workspace && git clone https://github.com/erdos-project/erdos.git
+RUN cd /home/erdos/workspace && git clone https://github.com/pschafhalter/erdos.git
 WORKDIR /home/erdos/workspace/erdos
 RUN cd /home/erdos/workspace/erdos
+RUN git checkout restructure
+
+# Install erdos.
 RUN cargo build --release
 # Install the python package
-RUN python3 python/setup.py install --user
+ENV PATH="/home/erdos/.local/bin:${PATH}"
```

Comment:
You seem to be setting the `PATH` twice?

Reply:
The first one sets up Cargo, this one sets up programs installed via pip (e.g. maturin).
I can combine them into 1 line, but I thought updating the `PATH` down here with the rest of the python logic made more sense.

Reply:
Apologies, I didn't see the append of the `$PATH` at the end. This should work fine!